#!/bin/sh -e
## 03_dll.conf_debian.dpatch by Julien BLACHE <jblache@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: dll.conf comments for Debian
## DP: Patch to the dll backend to look for pieces of dll.conf inside the
## DP: /etc/sane.d/dll.d/ directory. This is a facility for packages providing
## DP: external backends (like libsane-extras, hpoj and hplip).

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad sane-backends-1.0.19~cvs20071213~/backend/dll.conf.in sane-backends-1.0.19~cvs20071213/backend/dll.conf.in
--- sane-backends-1.0.19~cvs20071213~/backend/dll.conf.in	2007-12-13 21:30:29.941481743 +0100
+++ sane-backends-1.0.19~cvs20071213/backend/dll.conf.in	2007-12-13 21:30:31.468481793 +0100
@@ -1,3 +1,10 @@
+# /etc/sane.d/dll.conf - Configuration file for the SANE dynamic backend loader
+#
+# On Debian systems, the dll backend will also look for pieces of configuration
+# in the /etc/sane.d/dll.d directory -- packages providing backends should drop
+# a config file similar to dll.conf in this directory.
+#
+
 # enable the next line if you want to allow access through the network:
 net
 abaton
@@ -46,7 +53,6 @@
 nec
 niash
 pie
-pint
 pixma
 plustek
 #plustek_pp
@@ -72,11 +78,4 @@
 #umax_pp
 umax1220u
 v4l
-#
-# The following backends are not included in the sane-backends distribution
-# If you want to use them, download them from their webpages and read their
-# documentation 
-#
-# HP OfficeJet backend homepage: http://hpoj.sf.net/
-# Uncomment the following line if hpoj is installed:
-#hpoj
+
